pkgs/by-name/gr/gradia/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  gtk4,
+  glib,
+  ninja,
+  meson,
+  stdenv,
+  python3,
+  libportal,
+  libadwaita,
+  wrapGAppsHook4,
+  fetchFromGitHub,
+  nix-update-script,
+  blueprint-compiler,
+  desktop-file-utils,
+  gobject-introspection,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gradia";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "AlexanderVanhee";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "gradia";
```
pname in source fetching is a discouraged pattern due to overlay stability considerations (among other things), see also https://github.com/NixOS/nixpkgs/issues/277994